proto/regen/ecocredit/v1/tx.proto

```diff
@@ -305,3 +326,95 @@ message MsgUpdateClassMetadata {
 
 // MsgUpdateClassMetadataResponse is the MsgUpdateClassMetadata response type.
 message MsgUpdateClassMetadataResponse {}
+
+// MsgAddCreditType is the Msg/AddCreditType request type.
+// The credit types are immutable and cannot be deleted.
+message MsgAddCreditType {
+  // credit_types is a list of new credit types to add
+  repeated CreditType credit_types = 1;
+
+  // root_address is the address of the caller.
+  // this MUST equal the address of the gov module for the tx to succeed.
+  string root_address = 2;
+}
```

Comment:
Given this is the message for add, I don't think we need to include "The credit types are immutable and cannot be deleted." Also this should either be plural (we are adding multiple credit types) or we should limit to one credit type at a time.

```suggestion
// MsgAddCreditTypes is the Msg/AddCreditTypes request type.
message MsgAddCreditTypes {
  // credit_types is a list of credit types to add.
  repeated CreditType credit_types = 1;

  // root_address is the address of the signer.
  // This MUST equal the address of the gov module for the tx to succeed.
  string root_address = 2;
}
```